### PR TITLE
Add a section on community polyfills and tooling

### LINKED ISSUE #60: How to convert package-lock.json to import maps?
This looks so cool! Are there any implementations of "package-lock.json -> package-name-map" translators yet? (Is that what I should be thinking of?)

(Edit by @domenic: at the time this comment was written the proposal was known as "package name maps". I've renamed the title to be "import maps", but left the body of subsequent comments alone. Sorry if that's confusing!)

### LINKED ISSUE #108: Using rollup to generate import-maps
Has anyone explored building a rollup plugin to generate `import-map` as an output ?

This would move us closer to an unprocessed shipped code. Also, help in some development workflows.

### PR DESCRIPTION
**Pull request by domenic**
Closes #60. Closes #108. We'll discuss in #146 instead!

### CODE REVIEW

**Comment by guybedford** [comment]
Nice, could you include es-module-shims (http://npmjs.org/package/es-module-shims)?

**Comment by domenic** [comment]
For sure. Maybe you could suggest some wording for it plus SystemJS? I'm not sure how best to distinguish them while paying them their full due, in a short-ish sentence each.

File: README.md [hunk 1]
```diff
@@ -873,6 +874,19 @@ Several implementer questions come up around how `import:` URLs are envisioned t
...
 
+## Community polyfills and tooling
+
+Several members of the community have been working on polyfills and tooling related to import maps. Here are the ones we know about:
+
+* [@import-maps/generate](https://www.npmjs.com/package/@import-maps/generate) generates an import map from your `yarn.lock`.
+* [@jsenv/node-module-import-map](https://www.npmjs.com/package/@jsenv/node-module-import-map) generates an import map from your `package.json` and `node_modules/` directories.
+* [importly](https://www.npmjs.com/package/importly) generates an import map from a config file.
+* [Built-in Module Demo (with Rollup)](https://glitch.com/edit/#!/rollup-built-in-modules) contains code for a [Rollup](https://rollupjs.org/) plugin that generates an import map, focused on built-in module polyfills.
+* [SystemJS](https://github.com/systemjs/systemjs) provides a polyfill for import maps using `<script type="system-importmap">`.
```
Comment (guybedford) [inline README.md]: Ok, how about the following - 

```suggestion
* [SystemJS](https://github.com/systemjs/systemjs) provides a polyfill-like workflow for using import maps in older browsers using the System module format and a `<script type="system-importmap">`.
* [es-module-shims](https://github.com/guybedford/es-module-shims) provides an import maps polyfill for browsers with basic ES modules support.
```

**Comment by user3232** [comment]
Hi folks,

I've created API/CLI for generating importmaps from packages runtime directories and scheme to universally name runtime packages, you can check it out here: https://github.com/user3232/importmap .

**Pull request merged**